modoboa_stats/management/commands/logparser.py

```diff
@@ -277,73 +283,164 @@ def year(self, month):
             return self.__year - 1
         return self.__year
 
-    def _parse_line(self, line):
-        """Parse a single log line.
+    def domain(self, mail_address):
```

Comment:
There is already something available in modoboa to split an email address: https://github.com/modoboa/modoboa/blob/master/modoboa/lib/email_utils.py#L221.